boards/msba2-common/Makefile

```diff
@@ -1,4 +1,4 @@
-MODULE = board
+MODULE = msba2-common
```

Comment:
I know. But it breaks compilation in certain circumstances.
I don't understand exactly why, but I think our build system calls "make -C" for every DIRS entry, which contain the modules.
Now, that make call compiles all sources into .o's and then takes *all* .o's of the module's bindir directory and archives them into module.a.

Now if multiple DIRS define the same module, as long as the "make -C" calls are called sequentially, the second make would just pick up the modules from the first.

If they're called in parallel, the archives might miss some objects.

So unless we fix this, no two DIRS can compile into the same .a.